src/test/reporter.ts

```diff
@@ -60,13 +60,15 @@ export interface TestResult {
   stdout: (string | Buffer)[];
   stderr: (string | Buffer)[];
 }
+export type FullResult = {
+  status: 'passed' | 'failed' | 'timedout' | 'interrupted';
```

Comment:
What happens if you call test.skip in the test body?

Reply:
It is skipped. How's that related?

Reply:
Oh I misread it. This timeout is for the entire test run, got it.